lighthouse-core/scripts/dogfood-lhci.sh

```diff
@@ -41,4 +41,8 @@ npm install -g @lhci/cli@0.3.x
 # Collect our LHCI results.
 lhci collect --staticDistDir=./dist/now/english/
 # Upload the results to our canary server.
-lhci upload --serverBaseUrl="$LHCI_CANARY_SERVER_URL" --token="$LHCI_CANARY_SERVER_TOKEN" --github-token="$BUNDLESIZE_GITHUB_TOKEN"
+lhci upload \
+  --serverBaseUrl="$LHCI_CANARY_SERVER_URL" \
+  --token="$LHCI_CANARY_SERVER_TOKEN" \
+  --github-token="$BUNDLESIZE_GITHUB_TOKEN" \
```

Comment:
hah just noticed this. funny that we just reuse.

admittedly all the GH tokens necessary within CI makes a mess so i'm not entirely surprised. :)